test/test.js

```diff
@@ -577,4 +577,121 @@ describe('finalhandler(req, res)', function () {
         })
     })
   })
+
+  describe('Errors with cause', function () {
+    it('should return Error message with 1 level cause trace', function (done) {
+      var err = new Error('foo', { cause: new Error('bar') })
+
+      var expectedError = `Error: foo
```

Comment:
I dont think we want tests which couple to the entire stack trace of mocha and this library. Similarly, we dont want the test to couple to the full html. Both of these mean even small unrelated changes can make this one brittle.